src/style.css

```diff
@@ -0,0 +1,23 @@
+*{
+  font-family: Arial;
+  font-size: 13px;
+}
```

Comment:
1. add space after selector
2. add spaces between different blocks of styles
3. don't use `*` selector for everything, it's usually used for `box-sizing` or something like that, any other styles write to the tags you need in this particular situation